### ISSUE TITLE
dashboard returns 404 on raspberry pi

### ISSUE THREAD

**Issue by karel1980** [containous/traefik#2227]
### What did you do?
- Install docker on a raspberry pi 3
- "install" docker-compose as described here: https://hub.docker.com/r/t0rx/rpi-docker-compose/
- Followed the instructions from https://docs.traefik.io/ (create docker-compose.yml & docker-compose up)
- curl http://localhost:8080 -> 302 Found => /dashboard/
- curl http://localhost:8080/dashboard/ -> 404

### What did you expect to see?

$ curl http://localhost:8080/dashboard/
```
<!doctype html>
<html ng-app="traefik">
  <head>
    <meta charset="utf-8">
    <title>Træfik</title>
...
```

=> *this is the result I am getting when running the example docker-compose.yml on my mac*

### What did you see instead?

404 Not found

=> The raspberry pi

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.3.8
Codename:     raclette
Go version:   go1.8.1
Built:        2017-09-08_10:05:47AM
OS/Arch:      linux/arm
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
none (dev null)
```

### If applicable, please paste the log output in debug mode (`--debug` switch)

```
time="2017-10-08T18:40:22Z" level=info msg="Traefik version v1.3.8 built on 2017-09-08_10:05:47AM" 
time="2017-10-08T18:40:22Z" level=info msg="Using TOML configuration file //traefik.toml" 
time="2017-10-08T18:40:22Z" level=debug msg="Global configuration loaded {"GraceTimeOut":10000000000,"Debug":true,"CheckNewVersion":true,"AccessLogsFile":"","TraefikLogsFile":"","LogLevel":"DEBUG","EntryPoints":{"http":{"Network":"","Address":":80","TLS":null,"Redirect":null,"Auth":null,"Compress":false}},"Cluster":null,"Constraints":[],"ACME":null,"DefaultEntryPoints":["http"],"ProvidersThrottleDuration":2000000000,"MaxIdleConnsPerHost":200,"IdleTimeout":180000000000,"InsecureSkipVerify":false,"Retry":null,"HealthCheck":{"Interval":30000000000},"Docker":{"Watch":true,"Filename":"","Constraints":null,"Endpoint":"unix:///var/run/docker.sock","Domain":"docker.localhost","TLS":null,"ExposedByDefault":true,"UseBindPortIP":false,"SwarmMode":false},"File":null,"Web":{"Address":":8080","CertFile":"","KeyFile":"","ReadOnly":false,"Statistics":null,"Metrics":null,"Path":"","Auth":null},"Marathon":null,"Consul":null,"ConsulCatalog":null,"Etcd":null,"Zookeeper":null,"Boltdb":null,"Kubernetes":null,"Mesos":null,"Eureka":null,"ECS":null,"Rancher":null,"DynamoDB":null}" 
time="2017-10-08T18:40:22Z" level=info msg="Preparing server http &{Network: Address::80 TLS:<nil> Redirect:<nil> Auth:<nil> Compress:false}" 
time="2017-10-08T18:40:22Z" level=info msg="Starting provider *docker.Provider {"Watch":true,"Filename":"","Constraints":null,"Endpoint":"unix:///var/run/docker.sock","Domain":"docker.localhost","TLS":null,"ExposedByDefault":true,"UseBindPortIP":false,"SwarmMode":false}" 
time="2017-10-08T18:40:22Z" level=info msg="Starting provider *server.WebProvider {"Address":":8080","CertFile":"","KeyFile":"","ReadOnly":false,"Statistics":null,"Metrics":null,"Path":"","Auth":null}" 
time="2017-10-08T18:40:22Z" level=info msg="Starting server on :80" 
time="2017-10-08T18:40:22Z" level=debug msg="Provider connection established with docker 17.05.0-ce (API 1.29)" 
time="2017-10-08T18:40:22Z" level=debug msg="Validation of load balancer method for backend backend-proxy-tmp failed: invalid load-balancing method ''. Using default method wrr." 
time="2017-10-08T18:40:22Z" level=debug msg="Configuration received from provider docker: {"backends":{"backend-proxy-tmp":{"servers":{"server-tmp_proxy_1":{"url":"http://172.20.0.2:80","weight":0}},"loadBalancer":{"method":"wrr"}}},"frontends":{"frontend-Host-proxy-tmp-docker-localhost":{"entryPoints":["http"],"backend":"backend-proxy-tmp","routes":{"route-frontend-Host-proxy-tmp-docker-localhost":{"rule":"Host:proxy.tmp.docker.localhost"}},"passHostHeader":true,"priority":0,"basicAuth":[]}}}" 
time="2017-10-08T18:40:22Z" level=debug msg="Last docker config received more than 2s, OK" 
time="2017-10-08T18:40:22Z" level=debug msg="Creating frontend frontend-Host-proxy-tmp-docker-localhost" 
time="2017-10-08T18:40:22Z" level=debug msg="Wiring frontend frontend-Host-proxy-tmp-docker-localhost to entryPoint http" 
time="2017-10-08T18:40:22Z" level=debug msg="Creating route route-frontend-Host-proxy-tmp-docker-localhost Host:proxy.tmp.docker.localhost" 
time="2017-10-08T18:40:22Z" level=debug msg="Creating backend backend-proxy-tmp" 
time="2017-10-08T18:40:22Z" level=debug msg="Creating load-balancer wrr" 
time="2017-10-08T18:40:22Z" level=debug msg="Creating server server-tmp_proxy_1 at http://172.20.0.2:80 with weight 0" 
time="2017-10-08T18:40:22Z" level=info msg="Server configuration reloaded on :80" 
```

**Comment by ldez** [comment]
Thanks for your interest in Traefik !

Every question in the issues means features less 😉.

The issue tracker is for reporting bugs and feature requests only.
For end-user related support questions, refer to one of the following:

- [Stack Overflow](https://stackoverflow.com/questions/tagged/traefik) using the "traefik" tag
- the [Traefik community Slack channel](https://traefik.herokuapp.com)

**Comment by karel1980** [comment]
@ldez This is genuinely not a question. I don't understand why it was closed so quickly.
I have tried reproducing with the hypriot/rpi-traefik images (all versions available on github starting with v1.2.3, all the way up to v1.3.8. Only the very latest one (v1.3.8 gave a 404).
Here's the script I used:
```
pi@raspberrypi:~/vervaeke.info $ cat test-all.sh 
for t in $(cat tags); do
	perl -pi -e 's/    image.*/    image: hypriot\/rpi-traefik:'$t'/' docker-compose.yml
	dcokru
	curl http://localhost:8080/dashboard/ > $t.curlout
done
```
where tags is just:
```
v1.2.3
v1.3.0
...
```

Traefik clearly does not behave as described in the documentation (at least not on the raspberry pi).
